ush/python/pygfs/task/atm_analysis.py

```diff
@@ -58,30 +55,88 @@ def __init__(self, config):
         # Extend task_config with local_dict
         self.task_config = AttrDict(**self.task_config, **local_dict)
 
+        # Create JEDI object
+        self.jedi = Jedi(self.task_config, yaml_name)
+
+    @logit(logger)
+    def initialize_jedi(self):
+        """Initialize JEDI application
+
+        This method will initialize a JEDI application used in the global atm analysis.
+        This includes:
+        - generating and saving JEDI YAML config
+        - linking the JEDI executable
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        ----------
+        None
+        """
+        super().initialize()
```

Comment:
```suggestion
```
`super().initialize()` has nothing in it.  It does not need to be called.

None of the `Task.py` methods (except `__init__.py`) needs to be called as they are simple base methods.  If we were to make `class Task` as an abstract base class `ABC`, we would have to enforce all children to explicitly **have** to concretize those methods.  We chose to not force that on developers.